frontend/src/views/HomeView.vue

```diff
@@ -77,11 +82,16 @@ export default {
         }
 
         await Promise.all([fetchTasks(), fetchTeams()])
+            .catch((err) => {
+                console.log(err);
+                if (axios.isAxiosError(err) && err.response.status == 401) isAuth.value = false
+            })
```

Comment:
```suggestion
                if (axios.isAxiosError(err) && err.response.status == 401) { 
                  isAuth.value = false 
                }
            })
```

if で囲む時にはなるべく `{ }` しとくほうが良いスタイルなので覚えておきましょう（MUST にするかはチーム次第ですが、つけておくのをダメだというチームは殆どないはずです）